cmd/goal/clerk.go

```diff
@@ -984,6 +986,10 @@ var dryrunRemoteCmd = &cobra.Command{
 		if err != nil {
 			reportErrorf("dryrun-remote: %s\n", err.Error())
 		}
+		if rawOutput {
+			fmt.Println(string(protocol.EncodeJSON(&resp)))
```

Comment:
elsewhere in this code we do `fmt.Fprintf(os.Stdout, ...)` instead of `fmt.Println`

Reply:
Fixed, thanks